test/e2e/installplan_e2e_test.go

```diff
@@ -3223,17 +3208,11 @@ var _ = Describe("Install Plan", func() {
 	When("an InstallPlan is created with no valid OperatorGroup present", func() {
 		var (
 			installPlanName string
-			ns              *corev1.Namespace
 		)
 
 		BeforeEach(func() {
-			ns = &corev1.Namespace{}
-			ns.SetName(genName("ns-"))
-
-			// Create a namespace
-			Eventually(func() error {
-				return ctx.Ctx().Client().Create(context.Background(), ns)
-			}, timeout, interval).Should(Succeed(), "could not create Namespace")
+			// Make sure there are no OGs in the namespace already
```

Comment:
OGs are created in the `generatedNamespace` before each test in this file but this particular test relies on a blank slate to work consistently. This is a quick-and-dirty way to make sure we get that clean slate.

Reply:
I understand the intention is to reuse one namespace all for tests under this umbrella but should you take care of the fact a new InstallPlan still being created for each test in the same namespace?

Reply:
Ignore this comment, the AfterEach should take care of this. So we are cool here.